.idea/dataSources/2713ed1e-46ee-4482-af43-45c7d5c53ab1.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
AFAIK this file also contains other details of the database, like its name, access credentials, host information, etc. This information is the same for all instances of the environment, and therefore is shipped.

If there is information about the _structure_ of the database in this file too, that's fine: once the env has been instantiated, it would have the same structure in all instances of the same environment. Same thing, so.

Reply:
Ok, cool.